src/commonMain/kotlin/exchange.dydx.abacus/state/v2/supervisor/AccountSupervisor.kt

```diff
@@ -912,7 +912,7 @@ internal open class AccountSupervisor(
             state?.transferStatuses,
             state?.restriction,
             state?.launchIncentive,
-            Compliance(state?.compliance?.geo, compliance.status, compliance.updatedAt, compliance.expiresAt),
```

Comment:
@rosepuppy based on when `didSetComplianceStatus` is called, it seems like we should be passing in the entire compliance object instead of just the non-geo fields. Double checking I am not missing something here though.